src/main/java/core/basesyntax/DataProcess.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataProcess {
+    private static final int OPERATION = 0;
+    private static final int FRUIT = 1;
+    private static final int NUMBER = 2;
+    private static final String COMMA = ",";
+    private static final char ADD = '+';
+    private static final char SUBTRACT = '-';
+
+    public HashMap<String, Integer> process(String[] data) {
+        HashMap<String, Integer> database = new HashMap<>();
+        for (String datum : data) {
+            String[] check = datum.split(COMMA);
+            if (check[OPERATION].equals(Operation.BALANCE.getOperation())) {
```

Comment:
we should separate our concerns, you parse incoming data, and apply it immediately. Let's create a model to store a fruit transaction, and return it in this method